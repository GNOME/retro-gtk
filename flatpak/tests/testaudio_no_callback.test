[Retro Reftest]
Path=/testaudio_no_callback
Core=/app/lib/libretro/testaudio_no_callback_libretro.so

[Frame 0]
Tests=Run;Video;
Video=testaudio_no_callback.png
