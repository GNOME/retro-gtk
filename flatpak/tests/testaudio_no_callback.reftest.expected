/testaudio_no_callback/Boot: OK
/testaudio_no_callback/Options: OK
/testaudio_no_callback/0/State Refresh: OK
/testaudio_no_callback/0/Run: OK
/testaudio_no_callback/0/Video: OK
